pytest_order/sorter.py

```diff
@@ -224,6 +224,21 @@ def items_from_class_label(self, label: str, item: Item) -> List[Item]:
                         items.append(self.node_ids[node_id])
         return items
 
+    def items_from_path_label(self, label: str, item: Item) -> List[Item]:
+        items = []
+        item_id = item.node_id
+        label_len = len(label)
+        for node_id in self.node_ids:
+            if node_id.count("::") == 2:
+                path_index = node_id.index("::")
+                if label.endswith('/') and label in node_id:  # directory ordering
```

Comment:
The `in` check here could be ambiguous - maybe `node_id.startswith(label)`? Or we need more checks here...

Reply:
I took the approach of checking `endswith()` in order to match any part of a directory's path, even if not fully specified (as it allows shorter and more concise specification in the decorator parameters).

Reply:
Yes, actually I did the same with `endswith`, which wouldn't work in your case, of course... given that it needs to end with a slash this may be ok (it would theoretically allow ambiguities, but the user would be responsible to avoid them).